opendj-packages/opendj-docker/bootstrap/setup.sh

```diff
@@ -19,7 +29,6 @@ if [ -d /opt/opendj/bootstrap/schema/ ]; then
 fi
 
 if [ -d /opt/opendj/bootstrap/data/ ]; then
-  echo "Loading initial data:"
   for file in /opt/opendj/bootstrap/data/*;  do
```

Comment:
please move allow-pre-encoded-passwords inside this place

Reply:
done 925897989377f69deb09787d75b8e5c26e6aea72